cmd/tailscale/cli/update.go

```diff
@@ -324,6 +328,63 @@ func updateDebianAptSourcesListBytes(was []byte, dstTrack string) (newContent []
 	return buf.Bytes(), nil
 }
 
+func (up *updater) updateArch() (err error) {
+	if os.Geteuid() != 0 {
+		return errors.New("must be root; use sudo")
+	}
+
+	defer func() {
+		if err != nil && !errors.Is(err, errUserAborted) {
+			err = fmt.Errorf(`%w; you can try updating using "pacman -Sy tailscale"`, err)
```

Comment:
Might be nice to recommend the exact same command line that we run below? I like the long form of options since it's documentation.